src/build/cudos-blast.md

```diff
@@ -0,0 +1,334 @@
+# Cudos Blast
+
+Cudos Blast is a Node.js CLI (command line interface) tool for working with the Cudos blockchain. You can scaffold, compile and test your **Rust** smart contracts. JavaScript and Rust testing is supported.
+Utilizing `blast.config.js` it provides a possibility for deploying and interacting with them on a specified network (local, test or public).
+By using this tool you can also spin up a local [`Cudos node`](https://github.com/CudoVentures/cudos-node) and interact with it.
+
+## Table of Contents
+
+* [Installation](#installation) 
+* [Help and version](#help-and-version) 
+* [Initializing a project](#initializing-a-project) 
+* [Compiling smart contracts](#compiling-smart-contracts) 
+* [Running Rust tests](#running-rust-tests) 
+* [Testing contracts with JavaScript](#testing-contracts-with-javascript) 
+* [Interacting with a Cudos node](#interacting-with-a-cudos-node) 
+  * [Starting a local node](#starting-a-local-node) 
+  * [Stopping a running local node](#stopping-a-running-local-node) 
+  * [Checking node status](#checking-node-status) 
+* [Deploying smart contracts, interacting with them and running custom script files](#deploying-smart-contracts-interacting-with-them-and-running-custom-script-files) 
+  * [Available functions in global context](#available-functions-in-global-context)
+* [Network](#network)
+  * [Testnet](#testnet)
+  * [Mainnet](#mainnet)
+* [Managing accounts](#managing-accounts) 
+  * [Listing local node accounts](#listing-local-node-accounts) 
+  * [Adding a new local node account](#adding-a-new-local-node-account) 
+  * [Removing an existing local node account](#removing-an-existing-local-node-account) 
+  * [Funding an existing local node account](#funding-an-existing-local-node-account) 
+
+## Installation
+
+Make sure you have [Node.js](https://nodejs.org/en/download/package-manager/) installed.  [Docker](https://docs.docker.com/engine/install) is also required.
+
+| Prerequisite   | Minimum version | Recommended version |
+| ---            | ---             | ---                 |
+| Node.js        | 12.5.0          | 16.10.0             |
+| npm            | 6.9.0           | 7.24.0              |
+| Docker engine  | 19.03.13        | 20.10.12            |
+| Docker compose | 1.27.4          | 1.29.2              |  
+
+> For Windows users we recommend using Windows Subsystem for Linux ([WSL](https://docs.microsoft.com/en-us/windows/wsl/install-manual#downloading-distros)).
+> To avoid permission issues with `WSL`, you may have to [change](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory) `npm` default directory. 
+
+Install Cudos Blast package:
```

Comment:
Documentation doesn’t state if user should install it Cudos Blast as package or globally. It would be good to start with 2 scenarios - one as cd rustContract, second to run globally and shortly explain the difference between those

Reply:
It is up to the user how to install it - locally or globally. All of the developers that are used to working with `npm` libraries are aware of the difference between both. If there are not they should double check this on `npm`. I don't believe that `cudos-blast` documentation is the right place for this.